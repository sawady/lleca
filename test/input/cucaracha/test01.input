fun main() {
  putChar(65)
}
