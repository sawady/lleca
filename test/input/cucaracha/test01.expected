Cons(
  Function(
    main,
    Nil,
    Unit,
    Cons(
      StmtCall(
        putChar,
        Cons(
          ExprConstNum(
            65
          ),
          Nil
        )
      ),
      Nil
    )
  ),
  Nil
)
